app/build.gradle

```diff
@@ -61,9 +61,9 @@ ext {
             eventbus : '2.4.0',
             circleimageview : '2.0.0',
             dagger : '2.0.2',
-            mockito : '2.7.0',
+            mockito : '1.10.19',
```

Comment:
The app originally used Version 1.10.19. I upgraded it to 2.7.0 and realized that this led to compile errors or test failures (can't remember exactly) so I reverted to the old version for now